rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/tests/savedFilters.spec.ts

```diff
@@ -0,0 +1,162 @@
+import { mount } from "@vue/test-utils";
+import SavedFilters from "../savedFilters.vue";
+import { Notification, MessageBox } from "uiv";
+
+jest.mock("@/library/rundeckService", () => ({
+  getRundeckContext: jest.fn().mockReturnValue({ projectName: "test" }),
+}));
+jest.mock("../../../../library/stores/ActivityFilterStore", () => ({
+  ActivityFilterStore: jest.fn().mockImplementation(() => ({
+    loadForProject: jest.fn().mockReturnValue({
+      filters: [
+        { filterName: "Filter 1" },
+        { filterName: "Filter 2" },
+        { filterName: "Filter 3" },
+      ],
+    }),
+    saveFilter: jest.fn().mockResolvedValue({ success: true }),
+    removeFilter: jest.fn().mockResolvedValue(undefined),
+  })),
+}));
+jest.mock("uiv", () => ({
+  MessageBox: {
+    confirm: jest.fn(),
+    prompt: jest.fn().mockResolvedValue({ value: "test" }),
+  },
+  Notification: {
+    notify: jest.fn(),
+  },
+}));
+describe("SavedFilters", () => {
+  let wrapper;
+  const eventBus = {
+    on: jest.fn(),
+    emit: jest.fn(),
+    off: jest.fn(),
+  };
+  beforeEach(async () => {
+    wrapper = mount(SavedFilters, {
+      props: {
+        hasQuery: true,
+        query: {},
+        eventBus,
+      },
+      global: {
+        mocks: {
+          $t: (msg) => msg,
+        },
+        stubs: {
+          btn: true,
+        },
+      },
+    });
+  });
```

Comment:
Please move the mount call to a method, as raised in previous PRs, to keep the tests cleaner.